examples/rocket/persist/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "persist"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+
+[dependencies]
+rocket = { version = "0.5.0-rc.1", features = ["json"] }
+serde = "1.0"
+sqlx = { version = "0.5", features = ["runtime-tokio-native-tls", "postgres"] }
+shuttle-service = { version = "0.4.2", features = ["sqlx-postgres", "secrets", "web-rocket"] }
```

```suggestion
rocket = { version = "0.5.0-rc.2", features = ["json"] }
serde = { version = "1.0", features = ["derive"] }
shuttle-service = { version = "0.4.2", features = ["persist", "web-rocket"] }
```

The `serde` one might be wrong :man_shrugging: